packages/docs/docs/self-hosting/config-settings.md

```diff
@@ -615,6 +615,12 @@ Limit for the rate at which auth requests can be sent to or processed by the ser
 
 **Default:** `60/minute`
 
+### defaultFhirQuota
+
+Limit for the total number of FHIR request that can be sent to ot processed by the server. For more default see the [Rate Limit docs](/docs/rate-limits)
+
+**Default:** `50000/minute`
```

Comment:
This shouldn't probably be the actual default value (e.g. `50000`) to show how it's specified in the config